docs/book/src/clusterctl/commands/alpha-rollout.md

```diff
@@ -21,14 +21,6 @@ Use the `restart` sub-command to force an immediate rollout. Note that rollout r
 clusterctl alpha rollout restart machinedeployment/my-md-0
 ```
 
-### Undo
```

Comment:
This seems unrelated to this PR

Reply:
Yeah. I forgot it in the other PR. I'll move the delta over